### PR TITLE
MINOR: KafkaConfig should not expose internal config when queried for non-internal values

### PR DESCRIPTION
**Pull request by kamalcph**
Side fix: Print the topic/broker/broker-loggers configurations in alphabetical/sorted order. This will be useful when the user supplies the '--all' flag.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by kamalcph** [comment]
@showuon @chia7712 
Please take a look when you get a chance.

File: core/src/main/scala/kafka/admin/ConfigCommand.scala [hunk 1]
```diff
@@ -580,7 +580,7 @@ object ConfigCommand extends Logging {
       .filter(entry => configSourceFilter match {
         case Some(configSource) => entry.source == configSource
         case None => true
-      }).toSeq
+      }).toSeq.sortBy(entry => entry.name())
```
Comment (kamalcph) [inline core/src/main/scala/kafka/admin/ConfigCommand.scala:582]: side fix to display the configs in sorted order.

**Comment by kamalcph** [comment]
Test failures are unrelated to this patch.

**Comment by kamalcph** [comment]
@showuon @mimaison 
ping for the review!

**Review by showuon — APPROVED** [review]
LGTM! Nice catch!

**Comment by showuon** [comment]
@kamalcph , could you help rebase with latest trunk to trigger the CI build again? Thanks.

**Comment by kamalcph** [comment]
@showuon 
Rebased with the trunk. Thanks for reviewing it!

**Comment by kamalcph** [comment]
@ijuma @showuon 
Can you merge this patch? Rebased with the trunk.

**Comment by ijuma** [comment]
@showuon Are you going to merge this one?

**Comment by kamalcph** [comment]
@ijuma @showuon 

can you please merge the patch?

**Review by mimaison — APPROVED** [review]
LGTM

**Pull request merged**